hr_attendance_report_theoretical_time/views/hr_employee_views.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record
+        id="recompute_employee_theoretical_attendance"
+        model="ir.actions.act_window"
+    >
+        <field name="name">Recompute Theoretical Attendance</field>
+        <field name="res_model">recompute.theoretical.attendance</field>
+        <field name="binding_model_id" ref="model_hr_employee" />
+        <field name="view_mode">form</field>
+        <field name="context">{'default_employee_ids': id, 'employee_ids': id, }</field>
```

Comment:
@pedrobaeza i am not sure if this is correct.

Reply:
Ok, I will wait for Pedro's response to this change before pushing again with the squashing of the administrative commits.

Reply:
No, it isn't. It's a x2m field, so it requires another nomenclature. Maybe `[id]`, or `[(4, id)]`, but it should be tried.